tools/astarte_import/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM elixir:1.8.2-slim as builder
```

Comment:
Can we take this chance to move away from slim images in builders? They make compilation time longer due to more packages being installed, and they don't make any sense as they're only builders.